src/user/screens/profile.screen.js

```diff
@@ -84,10 +80,11 @@ class Profile extends Component {
       isPendingUser,
       isPendingOrgs,
       isPendingCheckFollowing,
-      navigation
+      navigation,
```

Comment:
I'm not too picky about dangling comments. On the contrary - I [prefer them](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8).

I think it would be good to come to a consensus and enforce whichever way we want in the linter